src/Docker.DotNet/Models/ContainerUpdateParameters.Generated.cs

```diff
@@ -117,7 +125,7 @@ public ContainerUpdateParameters(UpdateConfig UpdateConfig)
         public bool? OomKillDisable { get; set; }
 
         [DataMember(Name = "PidsLimit", EmitDefaultValue = false)]
-        public long PidsLimit { get; set; }
+        public long? PidsLimit { get; set; }
```

Comment:
This is a breaking change right?

Reply:
Looks like it is to me. Unfortunately, I do not see what options we have since this reflects changes made in the docker API.

Reply:
Ok thats fine. It needs to be null anyways otherwise its default of 0 will not actually set to unlimited as it will not be serialized in the first place.